implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/internal/LogbackMetricsSuppressingManySink.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2017 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.statsd.internal;
+
+import io.micrometer.core.instrument.binder.logging.LogbackMetrics;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Sinks;
+
+public class LogbackMetricsSuppressingManySink implements Sinks.Many<String> {
+    private final Sinks.Many<String> delegate;
+
+    public LogbackMetricsSuppressingManySink(Sinks.Many<String> delegate) {
+        this.delegate = delegate;
+    }
+
+    @Override
+    public Sinks.EmitResult tryEmitNext(String s) {
+        return delegate.tryEmitNext(s);
```

Comment:
not sure what `ignoreMetrics` does, but maybe consider wrapping `tryEmit` API too? or emit an error if the intent is to not use the `tryEmit*` methods at all?

Reply:
Wrapping with returning a result would require additional changes, which is why I opted to not do it. We currently aren't using the try methods and this is an internal class, but that is a good suggestion to throw an error so the intent is more explicit. Thanks

Reply:
I've gone with throwing an `UnsupportedOperationException` since this is an internal class and we have no usage of the try methods currently. If we have a use, we can consider properly implementing them in the future, I think. I've also added a JavaDoc description to the class.